modules/customize-certificates-add-service-serving.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
```

Comment:
We need to also document that process for annotating a configmap to get the CA bundle. This can be taken from the service-ca-operator repo README.

Reply:
These instructions have been included immediately below the instructions on adding the service certificates.